code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1259,9 +1259,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		var/hungry = (500 - H.nutrition) / 5 // So overeat would be 100 and default level would be 80
 		if(health_deficiency >= 40)
 			if(flight)
-				. += (health_deficiency / 75)
+				. += ((health_deficiency-39) / 75) // CIT CHANGE - adds -39 to health deficiency penalty to make the transition to low health movement a little less jarring
```

Comment:
-39/75.....

Reply:
Only values above 40 will have the -39 applied to them. its impossible to have the -39 applied with 0 health deficiency